tests/unit/store/state.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('state', () => {
   it('projectNumber should be required, visible and valid', () => {
     expect(state.orderFormFields[0].id).toEqual('projectNumber')
     // @ts-ignore
-    expect(state.orderFormFields[0].required()).toBeTruthy()
+    expect(state.orderFormFields[0].required()).toBeFalsy()
```

Comment:
this would mean we would also update the spec right ? 'projectNumber should not be required, visible and valid' or even 'projectNumber should only be required on submit' and 'the projectNumber should not be required on save'